src/simulators/matrix_product_state/matrix_product_state_internal.hpp

```diff
@@ -238,26 +213,20 @@ class MPS{
     return enable_gate_opt_;
   }
 
-  //  void store_measure(const AER::reg_t outcome, const AER::reg_t &cmemory, const AER::reg_t &cregister) const{
-  //           cout << " store_measure not supported yet" <<endl;}
-
-  double norm(const uint_t qubit, cvector_t &vmat) const {
+  double norm(const uint_t qubit, cvector_t &vmat) {
```

Comment:
Can the function and `vmat` arguments be `const` ?

Reply:
Yes. Done.